src/hooks/useMediaQuery/useMediaQuery.test.tsx

```diff
@@ -0,0 +1,10 @@
+import { renderHook } from '@testing-library/react';
+import { useMediaQuery } from './useMediaQuery.js';
+
+describe('useMediaQuery', () => {
```

Comment:
The hook has been used in a few projects already, it'd be nice to have but it's not necessary imo.

Reply:
Until we need to fix/change something :D

Reply:
It appears this is not possible with `@testing-library` because the DOM is never actually rendered. We'd need to test this using cypress/playwright or something, I think that's a bit out of scope for this PR but I'll make a ticket for this feature. #57